Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -4814,9 +4814,14 @@ bool isNodeFocused(Node& node)
 
 // DOM component of hidden definition.
 // https://www.w3.org/TR/wai-aria/#dfn-hidden
+bool isDOMHidden(const RenderStyle& style)
+{
+    return style.display() == DisplayType::None || style.usedVisibility() != Visibility::Visible;
+}
+
 bool isDOMHidden(const RenderStyle* style)
```

Comment:
Is this DOM hidden? Or is it RenderHidden?

Reply:
Yeah, `isRenderHidden` is probably more accurate here. I've renamed the function to that in the latest commit, thanks!